openapi/paths/storefront/payment-instruments@{id}@setup.yaml

```
@@ -5,18 +5,18 @@ get:
     - Storefront
   tags:
     - Payment instruments
-  summary: Retrieve payment instrument setup transaction
+  summary: Retrieve a payment instrument set up transaction
```

```suggestion
  summary: Retrieve a payment instrument setup transaction
```
It's a transaction type like authorize, capture, named: `setup`.

---

Updated in [005ee09](https://github.com/Rebilly/api-definitions/pull/1166/commits/005ee0916f6d0b5b02926de0c48739f2d1ce17b7)